shared/cfg/guest-os/Windows.cfg

```diff
@@ -314,3 +314,24 @@
             install_path = "C:\Program Files (x86)\JAM Software\HeavyLoad"
         i386,i686:
             install_path = "C:\Program Files\JAM Software\HeavyLoad"
+    driver_load:
+        driver_id_pattern = "(.*?):"
+        driver_load_cmd = C:\devcon.exe enable @DRIVER_ID
+        driver_unload_cmd = C:\devcon.exe disable @DRIVER_ID
+        x86_64:
+            devcon = "xcopy %s:\devcon\wnet_amd64\devcon.exe C: /Y"
+        i386:
+            devcon = "xcopy %s:\devcon\wnet_x86\devcon.exe C: /Y"
+        variants:
+            - with_nic:
```

Comment:
variants should  be in xx-test.cfg,  you can special guest related cfg like: driver_load..with_nic:  xxxxx